src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/AsylumCaseNotificationApiSender.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.iacaseapi.infrastructure.clients;
+
+import static java.util.Objects.requireNonNull;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iacaseapi.domain.service.NotificationSender;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.security.UserCredentialsProvider;
+
+@Service
+public class AsylumCaseNotificationApiSender implements NotificationSender<AsylumCase> {
+
+    private static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
```

Comment:
Think we've go this is quite a few places now, I'll add to the tech debt to centralize somewhere appropriate

Reply:
👍